extensions/wikia/ContentReview/ContentReview.i18n.php

```diff
@@ -59,6 +59,10 @@
 	'content-review-status-in-review' => 'In review',
 	'content-review-status-approved' => 'Approved',
 	'content-review-status-rejected' => 'Rejected,',
+
+	'content-review-feedback-link-text' => 'Provide feedback',
+	'content-review-rejection-explanation' => '==Submitted script change was rejected==
```

Comment:
Do you think we could add an `<a>` anchor with a revision's ID so that we could link directly to it from a right rail module? That would be great!